poetry.lock

```diff
@@ -0,0 +1,1340 @@
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
do we really need this 1000-line files to declare dependencies? I may not be up to date with the current Python approach, but wouldn't a simple requirements.txt for use with pip be enough?

Reply:
I think we don't need this file, it had been uploaded by accident.

Reply:
Deleted.